packages/components/mention/src/mention.ts

```diff
@@ -0,0 +1,118 @@
+import {
+  buildProps,
+  definePropType,
+  isFunction,
+  isString,
+} from '@element-plus/utils'
+import { UPDATE_MODEL_EVENT } from '@element-plus/constants'
+import { inputProps } from '@element-plus/components/input'
+import { filterOption } from './helper'
+
+import type { ExtractPropTypes } from 'vue'
+import type Mention from './mention.vue'
+import type { MentionOption } from './types'
+import type { Options } from '@element-plus/components/popper'
+
+export const mentionProps = buildProps({
+  ...inputProps,
+  /**
+   * @description mention options list
+   */
+  options: {
+    type: definePropType<MentionOption[]>(Array),
+    default: () => [],
+  },
+  /**
+   * @description prefix character to trigger mentions. The string length must be exactly 1.
+   */
+  prefix: {
+    type: definePropType<string | string[]>([String, Array]),
+    default: '@',
+  },
+  /**
+   * @description character to split mentions. The string length must be exactly 1.
+   */
+  split: {
```

Comment:
The documentation and comments state that the length limit is 1, and perhaps a validation function should be added.